### PR TITLE
fix: fix model sizes in supported models lists

### PR DESCRIPTION
**Pull request by joein**

### CODE REVIEW

**Comment by joein** [comment]
Hey @NirantK @Anush008 

I went through list of supported models and found some discrepancies.
I tried to fix them, could you please check if I am right about this fixes, or not?

There were various cases, I'll try to describe how I chose the sizes:

1. When there is only model.onnx available - I simply took its size
2. When there are model.onnx and model.onnx_data - I chose the latter, since the former one is usually less then 1mb
3. When there are model.onnx, modelfp16.onnx, model_quantized.onnx - I chose the first one, I assumed that it would be the right thing to choose because there are no mentions of fp16 / quantized, etc. in the descriptions, and I assumed that it takes model.onnx by default.

`BAAI/bge-small-en-v1.5` contained 2 sources - GCP and HF, however the models there varies in size. HF one is 2 times smaller then GCP, so I decoupled them.

intfloat/multilingual-e5-large had a wrong link, it was named differently in the cloud, I renamed it there.

File: fastembed/text/onnx_embedding.py [hunk 1]
```diff
@@ -72,19 +72,27 @@
...
         "sources": {
             "url": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-small-en-v1.5.tar.gz",
+        },
+    },
+    {
+        "model": "BAAI/bge-small-en-v1.5-quantized",
+        "dim": 384,
+        "description": "Fast English model",
+        "size_in_GB": 0.06,
+        "sources": {
```
Comment (Anush008) [inline fastembed/text/onnx_embedding.py:82]: https://github.com/qdrant/fastembed/blob/main/docs%2Fexamples%2FSupported_Models.ipynb should be rerun to include this.

Comment (joein) [reply fastembed/text/onnx_embedding.py:82]: I'll do it, thank you :)

**Review by Anush008 — APPROVED** [review]

**Review by NirantK — APPROVED** [review]

**Comment by NirantK** [comment]
Wherever we've quantized models which meet the allclose tests — we'll use that as our default, without supporting the direct onnx import to avoid confusion. 

We also prefer HF over URL download.

The split here would've changed the underlying default model without informing the user, so I've tried to make that consistent for all models [here](https://github.com/qdrant/fastembed/pull/167/commits/985ad20d1021acb6c80592ed3086c7c67be9ad99)

cc @joein @Anush008

File: fastembed/text/onnx_embedding.py [hunk 2]
```diff
@@ -70,15 +61,6 @@
         "dim": 384,
         "description": "Fast and Default English model",
         "size_in_GB": 0.13,
```
Comment (joein) [inline fastembed/text/onnx_embedding.py:63]: I believe this should be adjusted since the quantized model is 2 times smaller

**Pull request merged**